deployment/.env.example

```diff
@@ -3,3 +3,6 @@ MQTT_PASSWORD=sample_password
 DOCKER_INFLUXDB_INIT_USERNAME=sample_user
 DOCKER_INFLUXDB_INIT_PASSWORD=sample_password
 DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=sample_token
+GF_SECURITY_ADMIN_USER=sample_user
```

Comment:
`GF` should means Grafana? If yes - then change to `GRAFANA` please

Reply:
These names are defined by Grafana so we cannot change them - https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables